enumerable.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    i = 0
+    while i < length
+      yield(self[i])
+      i += 1
+    end
+  end
```

Comment:
* [ ] If no block is given (i.e check with block_given? at the beginning of the method) the method should return an Enumerator.